common/include/pcl/conversions.h

```diff
@@ -169,6 +169,13 @@ namespace pcl
     cloud.height   = msg.height;
     cloud.is_dense = msg.is_dense == 1;
 
+    // check if there is data to copy
+    if (msg.data.empty())
```

Comment:
`msg.data` should not be used in this function, but `msg_data`.
However, I would say `msg.width*msg.height == 0` is the best check to determine whether `cloud[0]` causes a problem, no?